pkg/integration/integration.go

```diff
@@ -66,7 +66,10 @@ func StartAPIServer(crdDirectoryPath string) (*rest.Config, *envtest.Environment
 // preventing us from integration testing webhooks, we omit this flag and instead enable
 // the AlwaysAdmit plugin which does not clobber the webhook plugin.
 func kubeAPIServerFlags() []string {
-	flags, found := []string{"--enable-admission-plugins=AlwaysAdmit"}, false
+	flags, found := []string{
+		"--disable-admission-plugins=ServiceAccount",
```

Comment:
We should leave a comment that explains why this is disabled. Will be good to reference our reasoning so people can update it in future, if necessary.

Reply:
I can't actually remember why we disabled this, other than we got an error of some sort. Do you?